packages/page-staking/src/Overview/SummaryNominators.tsx

```diff
@@ -0,0 +1,93 @@
+// Copyright 2017-2022 @polkadot/app-staking authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+
+import { Card, CardSummary, SummaryBox } from '@polkadot/react-components';
+import { FormatBalance } from '@polkadot/react-query';
+import { formatNumber } from '@polkadot/util';
+
+import { useTranslation } from '../translate';
+import { SortedTargets } from '../types';
+import { Section, Title } from './index';
+import SpinnerWrap from './SpinnerWrap';
+
+interface Props {
+  targets: SortedTargets;
+}
+
+function SummaryNominators ({ targets: { maxNominatorsCount,
+  minNominatorBond,
+  nominatorActiveCount,
+  nominatorElectingCount,
+  nominatorMinActiveThreshold } }: Props) {
+  const { t } = useTranslation();
+
+  return (
```

Comment:
we need an info banner here with a short text that says: 

"In order to receive staking rewards, and be exposed with an active validator to slash, a nominator needs to be 'active'. Compare the below thresholds with your active stake to ensure you are in the correct set between 'active' and 'intention'.

Reply:
Ideally, we could link to polkadot wiki or at least to an edition of my staking newsletter where these roles are explained, but I supposed that might be against your policy @jacogr? LMK what you think. This is the current index of the staking updates that I compile every month, and the February one is particularly useful for this page. https://gist.github.com/kianenigma/aa835946455b9a3f167821b9d05ba376

Reply:
Linking is all fine - there are a number of links already, including to the wiki.

Reply:
nit: I think the box should go inside `nominators` box 🙈